plugin/src/ios/AppLovinMAX.m

```diff
@@ -319,6 +326,119 @@ - (void)setTestDeviceAdvertisingIds:(CDVInvokedUrlCommand *)command
     [self sendOKPluginResultForCommand: command];
 }
 
+-(void)setYearOfBirth:(CDVInvokedUrlCommand *)command
+{
+    NSNumber *yearOfBirth = [command argumentAtIndex: 0];
+    if ( [self isPluginInitialized] )
+    {
+        self.sdk.targetingData.yearOfBirth = yearOfBirth;
+        self.yearOfBirthToSet = nil;
+    }
+    else
+    {
+        self.yearOfBirthToSet = yearOfBirth;
+    }
+    
+    [self sendOKPluginResultForCommand: command];
+}
+
+-(void)setGender:(CDVInvokedUrlCommand *)command
+{
+    NSNumber *gender = [command argumentAtIndex: 0];
+    if ( [self isPluginInitialized] )
+    {
+        self.sdk.targetingData.gender = [gender intValue];
+        self.genderToSet = nil;
+    }
+    else
+    {
+        self.genderToSet = gender;
+    }
+    
+    [self sendOKPluginResultForCommand: command];
+}
+
+-(void)setMaximumAdContentRating:(CDVInvokedUrlCommand *)command
+{
+    NSNumber *contentRating = [command argumentAtIndex: 0];
+    if ( [self isPluginInitialized] )
+    {
+        self.sdk.targetingData.maximumAdContentRating = [contentRating intValue];
+        self.contentRatingToSet = nil;
+    }
+    else
+    {
+        self.contentRatingToSet = contentRating;
+    }
+    
+    [self sendOKPluginResultForCommand: command];
+}
+
+-(void)setEmail:(CDVInvokedUrlCommand *)command
+{
+    NSString *email = [command argumentAtIndex: 0];
+    if ( [self isPluginInitialized] )
+    {
+        self.sdk.targetingData.email = email;
+        self.emailToSet = nil;
+    }
+    else
+    {
+        self.emailToSet = email;
+    }
+    
+    [self sendOKPluginResultForCommand: command];
+}
+
+-(void)setPhoneNumber:(CDVInvokedUrlCommand *)command
+{
+    NSString *phoneNumber = [command argumentAtIndex: 0];
+    if ( [self isPluginInitialized] )
+    {
+        self.sdk.targetingData.phoneNumber = phoneNumber;
+        self.phoneNumberToSet = nil;
+    }
+    else
+    {
+        self.phoneNumberToSet = phoneNumber;
+    }
+    
+    [self sendOKPluginResultForCommand: command];
+}
+
+- (void)setKeywords:(CDVInvokedUrlCommand *)command
+{
+    NSArray<NSString *> *keywords = [command argumentAtIndex: 0];
+    
+    if ( [self isPluginInitialized] )
```

Comment:
Can we keep spacing of the `[command argumentAtIndex: 0]` call to `if ( [self isPluginInitialized] )` consistent across these new methods? I think I'd prefer a new line to break them up.